lib/movie_card/movie_actions.dart

```diff
@@ -0,0 +1,109 @@
+import 'package:flutter/material.dart';
+import '../movie.dart';
+import 'movie_like_counter.dart';
+import 'movie_secondary_button.dart';
+
+class MovieActions extends StatelessWidget {
+  const MovieActions({super.key, required this.movie, required this.counter});
+
+  final Movie movie;
+  final int counter;
+  static const int counterFlex = 1;
+  static const int buttonsBarFlex = 3;
+  static const int mainActionFlex = 3;
+  static const double mainActionPadding = 10;
+  static const double avgVotesFontSize = 12;
+
+  @override
+  Widget build(BuildContext context) {
+    final String averageRating = 'Avg. vote: ${movie.voteAverage}';
```

Comment:
the first part of the string can go outside of the build method as a static const and the concatenation can be done where it is used